examples/bri-3/src/bri/workgroup/workgroups/models/workgroup.ts

```diff
@@ -37,6 +44,7 @@ export class Workgroup {
     participants: BpiSubject[],
     worksteps: Workstep[],
     workflows: Workflow[],
+    status: WorkgroupStatus,
```

Comment:
I think there is no need to be given in the ctor. When created, workgroup is automatically set to Active.